src/GitVersionCore/Extensions/GitVersionOptionsExtensions.cs

```diff
@@ -22,20 +20,11 @@ public static string GetDotGitDirectory(this GitVersionOptions gitVersionOptions
                 : dotGitDirectory;
         }
 
-        public static string GetProjectRootDirectory(this GitVersionOptions gitVersionOptions)
+        public static string GetRepositoryWorkingDirectory(this GitVersionOptions gitVersionOptions)
         {
-            if (!string.IsNullOrWhiteSpace(gitVersionOptions.DynamicGitRepositoryPath))
-            {
-                return gitVersionOptions.WorkingDirectory;
-            }
-
-            var dotGitDirectory = Repository.Discover(gitVersionOptions.WorkingDirectory);
-
-            if (string.IsNullOrEmpty(dotGitDirectory))
-                throw new DirectoryNotFoundException($"Can't find the .git directory in {dotGitDirectory}");
-
-            using var repository = new Repository(dotGitDirectory);
-            return repository.Info.WorkingDirectory;
+            //return gitVersionOptions.WorkingDirectory;
```

Comment:
Thought it made more sense to always ask git what the working directory is, rather than assume its the cli 's working directory (current directory) - given we always resolve the dotGitDirectory.